vslib/src/sai_vs_policer.cpp

```diff
@@ -40,6 +40,20 @@ sai_status_t vs_clear_policer_stats(
     return SAI_STATUS_NOT_IMPLEMENTED;
 }
 
+sai_status_t vs_get_policer_stats_ext(
```

Comment:
this one should replace vs_sai_get_policer_stats_ext

Please remove vs_sai_get_policer_stats_ext above.